lib/internal/fs/utils.js

```diff
@@ -729,6 +738,18 @@ const defaultRmdirOptions = {
   recursive: false,
 };
 
+const validateCopyOptions = hideStackFrames((options) => {
+  options = { ...defaultCopyOptions, ...options };
+  validateBoolean(options.dereference, 'options.dereference');
+  validateBoolean(options.errorOnExist, 'options.errorOnExist');
+  validateBoolean(options.force, 'options.force');
+  validateBoolean(options.preserveTimestamps, 'options.preserveTimestamps');
+  if (options.filter) {
```

Comment:
```suggestion
  if (options.filter !== undefined) {
```

Otherwise it's the only option to allow other falsy values.

Reply:
Well, actually other options do not even allow to pass `undefined`. This might be surprising for users.

Reply:
@targos should I opt for:

`if (options.hasOwnproperty('filter')) {}`

Reply:
```suggestion
  if ('filter' in options) {
```

Reply:
I see that the validation of `fs.rm` options has the same issue. For now, I'd go for `!== undefined` or `!= null`